pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -263,6 +263,8 @@ spec:
   # should be cleared or retained.
   - name: RetainReplicaData
     enabled: "false"
+  - name: ReplicaCount
```

Comment:
Why this required?

Reply:
I see that for deletion, you are iterating through the count. 

Can you also verify the following test case please:
- Create a jiva volume with replica count=1 
- Delete it

In other words, I am wondering if during delete, we really need to know the replica count. Can we not make use of the list replica deployments and then use that resource to delete.

Reply:
No the ReplicaCount config is not required. I will remove it.
The iteration while deleting is done on the names of the listed deployments.
```yaml
apiVersion: openebs.io/v1alpha1
kind: RunTask
metadata:
  name: jiva-volume-delete-deletereplicadeployment-default
spec:
  meta: |
    {{- $jivapodsns := .TaskResult.jivapodsinopenebsns.ns | default .Volume.runNamespace -}}
    {{- $replicaDeployList := .TaskResult.deletelistrep.names | split " " -}}
    id: deletedeleterep
    runNamespace: {{ $jivapodsns }}
    apiVersion: apps/v1
    kind: Deployment
    action: delete
    repeatWith:
      metas:
      {{- range $k, $v := $replicaDeployList }}
      - objectName: {{ $v | quote }}
      {{- end }}        
```
I have tested it for 1 and 3 replicas and it works fine.

Reply:
It was required earlier for the check that has been removed.